shopfloor/models/stock_quant_package.py

```diff
@@ -15,6 +15,7 @@ class StockQuantPackage(models.Model):
     planned_move_line_ids = fields.One2many(
         comodel_name="stock.move.line",
         inverse_name="result_package_id",
+        domain=[("state", "not in", ("done", "cancel"))],
```

Comment:
For performance we should compare the state to open ones (`confirmed`, `assigned`, etc...) as the number of move lines `done` and `cancel` will increase over time, that doesn't help PostgreSQL.